src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -139,30 +140,34 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
                                 if (printPackages)
                                 {
                                     var hasAutoReference = false;
-                                    ProjectPackagesPrintUtility.PrintPackages(packages, projectName, listPackageArgs, ref hasAutoReference);
-                                    autoReferenceFound = autoReferenceFound || hasAutoReference;
+                                    List<ListPackageReportFrameworkPackage> projectFrameworkPackages = ProjectPackagesPrintUtility.GetPackagesMetaData(packages, listPackageArgs, ref hasAutoReference);
```

Comment:
naming: `GetPackagesMetadata`. 

Turns out there were multiple places with this :D